apps/mark-scan/backend/src/app.test.ts

```diff
@@ -240,3 +254,40 @@ test('configureWithSampleBallotPackageForIntegrationTest configures electionGene
   const electionDefinitionResult = await apiClient.getElectionDefinition();
   expect(electionDefinitionResult).toEqual(electionGeneralDefinition);
 });
+
+test('installs UI Strings API', async () => {
```

Comment:
I'm taking a sec to wrap my mind around the difference between these tests and the tests in `app.ui_strings.test.ts`, specifically trying to understand why the former are necessary.

It seems like the main thing tested in this file that's not tested in `app.ui_strings.test.ts` is the Grout client's ability to access the endpoints. Otherwise, `app.ui_strings.test.ts` tests everything tested in here and more (installation of the UI strings API + actual implementation since it doesn't mock).

I wonder if we can just assume that the Grout client will work fine, given `libs/grout` tests. Or alternatively, have the shared `ui_strings_api_test_runner.ts` create a Grout client for testing.

I might be missing something! Just wondering if we can get rid of these `app.test.ts` tests so that everything, tests included, really is automatic.

Reply:
Ah, thanks - I lost perspective on this after a couple iterations. Initially had the shared test runner just taking in a `store` input and testing that, but you're spot on WRT not needing these tests since the test runner is testing against an app's actual API.
Will go ahead and drop these!